cfme/tests/infrastructure/test_quota_tagging.py

```diff
@@ -120,3 +148,39 @@ def test_quota_tagging(appliance, provider, setup_provider, set_entity_quota_tag
     provision_request = appliance.collections.requests.instantiate(request_description)
     provision_request.wait_for_request(method='ui')
     assert provision_request.row.reason.text == "Quota Exceeded"
+
+    @request.addfinalizer
+    def delete():
+        provision_request.remove_request()
```

Comment:
**REQUIRED**
You should place it before assertion and please make it slightly nicer:
```python
request.addfinalizer(provision_request.remove_request)
```